R/zoop_downloader.R

```diff
@@ -400,14 +541,15 @@ Zoopdownloader <- function(
       dplyr::mutate(Source="FRP", #add variable for data source
                     SizeClass="Meso",
                     Microcystis = dplyr::recode(.data$Microcystis, `1=absent`="1", `2=low`="2"))%>%
-      dplyr::select(.data$Source, .data$Date, .data$Datetime,
+      dplyr::select(.data$Source, .data$Date, .data$Datetime, .data$Latitude, .data$Longitude,
```

Comment:
Would these be different coordinates than we had in the stations file previously? Are they the exact locations of each sample instead of the general locations of each station? If so, is there a benefit to doing it this way instead of using the general location of each station?

Reply:
It looks like this is the exactly location of the trawl, rather than the general location of the station. Personally, I think it's more accurate to show exactly where the trawl was taken rather than the general location, but I'm willing to be overrruled there.

Reply:
OK, I think that's fine. We'll just want to be clear in the news file that we're making this change